invenio_accounts/admin.py

```diff
@@ -129,13 +134,14 @@ class RoleView(ModelView):
     list_all = ('id', 'name', 'description')
 
     column_list = \
-        form_columns = \
         column_searchable_list = \
         column_filters = \
```

Comment:
The password is only added to `form_columns` so that it is editable but it will not be displayed in the list view or be searchable.